client/models/lab_tests_model.ts

```diff
@@ -17,6 +17,16 @@ export interface PatientDetails {
   last_name: string;
 }
 
+export interface HealthInsurance {
+  front_image: Image;
+  back_image: Image;
+}
+
+interface Image {
```

Comment:
Needs to be exported too?

Reply:
Not really, but yep let's export it for completeness